### ISSUE TITLE
New mail notification triggers even for filtered mail

### ISSUE THREAD

**Issue by bondolo** [roundcube/roundcubemail#7890]
Using 1.3.16 (sonic.net customer). Thank you for providing an excellent webmail client!

I have filters set up to send mailing list and other mail not addressed to me to a separate mailbox. If I am viewing the inbox I still get browser notifications for incoming mail messages that are filtered to other mailboxes. Notifications should only happen for messages arriving (after filtering) in to the current mailbox.

**Comment by alecpl** [comment]
I do not agree. If you set `check_all_folders=true` all folders are being checked for a new mail. The same set of folders is used by the plugin. So, you can switch to only INBOX by disabling this option.

I understand this might not be the same. Maybe there should be a configuration option/user preference.
